vanagon.gemspec

```diff
@@ -12,6 +12,8 @@ Gem::Specification.new do |gem|
   gem.authors  = ['Puppet Labs']
   gem.email    = 'info@puppetlabs.com'
   gem.homepage = 'http://github.com/puppetlabs/vanagon'
+  gem.specification_version = 3
+  gem.required_ruby_version = '~> 2.1.0'
```

Comment:
Do we want this this tightly bound or do we expect it to work with any 2.x >= 2.1.0?